CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # Change Log
 
+## 0.5.0 - 2020-12-01
+
+### Added
+
+- Publish local projects directly to GitHub
+- Advanced creation to have more control over creation
+- Download remote app settings
+- Notification for when long running GitHub actions complete
+
+### Changed
+- Functions/App Setting tree items no longer appear if build doesn't contain functions
+- `Create New Static Web App` prompts now asks for deployment method
```

Comment:
Should "prompts" be singular here?
```suggestion
- `Create New Static Web App` prompt now asks for deployment method
```

Or should "asks" be singular?
```suggestion
- `Create New Static Web App` prompts now ask for deployment method
```

Reply:
Prompts should probably be plural in this case, but I think I should probably just use the word "wizard" since that encompasses the entirety of the prompts?

Reply:
Oh I see why you were asking now heh